Rubberduck.Main/Root/RubberduckIoCInstaller.cs

```diff
@@ -244,8 +244,7 @@ private void RegisterQuickFixes(IWindsorContainer container, Assembly[] assembli
                 container.Register(Classes.FromAssembly(assembly)
                     .IncludeNonPublicTypes()
                     .Where(type => type.IsBasedOn(typeof(IQuickFix)) && type.NotDisabledExperimental(_initialSettings))
-                    .WithService.Base()
-                    .WithService.Self()
```

Comment:
The `Self()` should work and is important at the places where specific quickfixes get injected.